src/Entity/OgMembership.php

```diff
@@ -482,6 +483,41 @@ public function save() {
     return $result;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  protected function invalidateTagsOnSave($update) {
+    parent::invalidateTagsOnSave($update);
+
+    // A membership was created or updated: invalidate the membership list cache
+    // tags of its group. An updated membership may start to appear in a group's
+    // membership listings because it now meets those listings' filtering
+    // requirements. A newly created membership may start to appear in listings
+    // because it did not exist before.
+    if ($group = $this->getGroup()) {
```

Comment:
There's no real consensus on this but I'm not a fan of assignments in conditionals
See https://www.drupal.org/project/coding_standards/issues/2907332

Reply:
OK! I will move the assignment out of the `if` statement.